dbms/CMakeLists.txt

```diff
@@ -184,6 +184,7 @@ target_link_libraries (tiflash_common_io
     prometheus-cpp::pull
     cpptoml
     magic_enum
+    kv_client
```

Comment:
why `tiflash_common_io` rely on `kv_client`?

Reply:
```
/CMakeFiles/tiflash_common_io.dir/src/IO/ChecksumBuffer.cpp.o -c /DATA/disk1/qiuyang/projects/tiflash/dbms/src/IO/ChecksumBuffer.cpp
In file included from /DATA/disk1/qiuyang/projects/tiflash/dbms/src/IO/ChecksumBuffer.cpp:15:
In file included from /DATA/disk1/qiuyang/projects/tiflash/dbms/src/IO/ChecksumBuffer.h:21:
In file included from /DATA/disk1/qiuyang/projects/tiflash/dbms/src/Encryption/FileProvider.h:20:
In file included from /DATA/disk1/qiuyang/projects/tiflash/dbms/src/Encryption/KeyManager.h:17:
In file included from /DATA/disk1/qiuyang/projects/tiflash/dbms/src/Encryption/EncryptionPath.h:18:
In file included from /DATA/disk1/qiuyang/projects/tiflash/contrib/client-c/include/pingcap/pd/IClient.h:3:
/DATA/disk1/qiuyang/projects/tiflash/contrib/client-c/include/pingcap/Config.h:7:10: fatal error: 'grpcpp/security/credentials.h' file not found
    7 | #include <grpcpp/security/credentials.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

may files in `dbms/src/IO` include files in `dbms/src/Encryption`

Reply:
I will open a new pr to refactor the directory later.

Reply:
We can wait for this PR https://github.com/tikv/client-c/pull/140 and then refine the include header in TiFlash. So using `KeyspaceID` won't need to rely on grpc and `tiflash_common_io` do not rely on `kv_client`